src/dotnet/commands/dotnet-ref/Program.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration;
+
+namespace Microsoft.DotNet.Tools.Ref
+{
+    public class RefCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            DirectoryInfo currDir = new DirectoryInfo(Directory.GetCurrentDirectory());
+            FileInfo projectFile = currDir.GetFiles("*.csproj").FirstOrDefault();
+
+            if (!projectFile.Exists)
```

Comment:
This should happen later. If this check happens here, a user can't execute the "help" command to figure out what is wrong.